Added coloredlogs to the SDK

Patrick-DS opened the pull request:
**Proposed changes**:
- *Added coloredlogs to the SDK.* I assume I just had to add it where the \_\_main__ gets called, since all the files are getting their logger from the line 'logger = logging.getLogger(\_\_name__)" so I just imported the utils from rasa and added the same lines that are in the rasa_core library. It worked for me.

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

Patrick-DS commented:
Actually it's not literally a dependency on rasa_core, rasa_core_sdk has the exact same utils file containing the exact same command. I just need to pull the function from the right import statement and it'll work the same.

akelad commented:
well, it is a dependency on rasa_core because you're importing from rasa_core :)

akelad commented:
Ah wait got it, yes just modify it to import from rasa_core_sdk instead

Patrick-DS commented:
Since my first commit was not quite correct, I did another one to add what was necessary and squashed the two commits (because it really is just one commit, I won't leave my mistakes in there). Do I need anything to ask for the PR to be reviewed again, or can you just approve it now?

I had to add the function I added in the utils of the SDK because that function was only in the rasa_core utils file and I needed it to add log levels, which are required by configure_colored_logging.
